build.sh

```diff
@@ -13,4 +13,4 @@ while [[ -h $source ]]; do
 done
 
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
-"$scriptroot/eng/build.sh" --build $@
\ No newline at end of file
+"$scriptroot/eng/build.sh" --build --solution  Roslyn.sln $@
```

Comment:
Could just change the default in `eng/build.sh` on line 78?

Reply:
This will cause CI to build Roslyn.sln, which I don't think we want.